client-src/elements/chromedash-activity-log.ts

```diff
@@ -72,11 +72,13 @@ export class ChromedashAmendment extends LitElement {
 customElements.define('chromedash-amendment', ChromedashAmendment);
 
 export interface User {
+  // TODO(markxiong0122): Move this interface to chromedash-app.js after migration.
```

Comment:
The interfaces should be next to the functions that create instances in the first place, which is in cs-client in this case.

```suggestion
  // TODO(markxiong0122): Move this interface near cs-client.js:getPermissions() after migration.
```